lib/adapters/product-adapters.ts

```
@@ -67,6 +67,69 @@ export function itemToUnifiedProduct(
+      } else if (item.manufacturingDate instanceof Date) {
+        // Handle Date objects directly
+        formattedManufacturingDate = item.manufacturingDate
+          .toISOString()
+          .split("T")[0];
+        console.log(
+          "Converted Date object to YYYY-MM-DD:",
+          formattedManufacturingDate
+        );
+      } else {
+        // Try to convert to Date and format
+        const date = new Date(item.manufacturingDate as any);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)